CodeConverter.Web/wwwroot/js/index.js

```diff
@@ -54,12 +53,28 @@ var app = new Vue({
                         output.select();
                     });
                 }
-            };
-            xhttp.open("POST", "/api/converter/", true);
-            xhttp.setRequestHeader("Content-Type", "application/json");
+            })
+            .catch(function (error) {
+                capturedObj.converterCallInflight = false;
```

Comment:
Good that we're catching errors now. We could have done so with `xhttp.onError` but never did.
Now that we are, it seems like a good opportunity to set `capturedObj.errorMessageOnResponse` to tell the user what's happened. Then just console.log any extra details not already shown in the UI.

In the XMLHttpRequest world we could have used onError for this.